### PR TITLE
UPDATE - index.ejs update

### PR DESCRIPTION
**Pull request by muise001**
Updated the book to the OBA's corporate identity. Added the side menu

### CODE REVIEW

File: src/js/modules/book.js [hunk 1]
```diff
@@ -6,8 +6,9 @@ const mainBook = {
     })
     document.getElementById('pageSlider').addEventListener('change', () => {
```
Comment (velomovies) [inline src/js/modules/book.js]: Using `getElementById`? Other times we use querySelectorAll and querySelector. We should use one of them...

File: src/styles/style.css [hunk 2]
```diff
@@ -34,9 +45,9 @@
 }
 
 .page {
-  background-image: url("https://img.freepik.com/free-photo/grunge-wall-texture_1194-5384.jpg?size=338&ext=jpg");
+  /* background-image: url("https://img.freepik.com/free-photo/grunge-wall-texture_1194-5384.jpg?size=338&ext=jpg"); */
```
Comment (velomovies) [inline src/styles/style.css]: Delete unused css

File: src/styles/style.css [hunk 3]
```diff
@@ -299,6 +298,7 @@ svg{
 
 #range input {
   /* transform: scale(3) rotate(270deg); */
+  /* For later use. Dont remove */
```
Comment (velomovies) [inline src/styles/style.css]: Hahaha nice comment!

**Comment by muise001** [comment]
thanks for the great review @velomovies 😸

**Pull request merged**